src/model/account/Account.ts

```diff
@@ -133,13 +133,22 @@ export class Account {
     }
 
     /**
-     * Sign raw data
+     * Sign raw string data
      * @param data - Data to be signed
      * @return {string} - Signed data result
      */
     public signData(data: string): string {
         return convert.uint8ToHex(KeyPair.sign(this.keyPair,
-                            convert.hexToUint8(convert.utf8ToHex(data)),
-                        ));
+            convert.hexToUint8(convert.utf8ToHex(data)),
+        ));
+    }
+
+    /**
+     * Sign raw binary data
+     * @param data - Data to be signed
+     * @return {string} - Signed data result
+     */
+    public signBinary(data: Uint8Array): string {
```

Comment:
In case of the input be a `Uint8Array`, maybe the return type should be the same?